sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesResult.cs

```diff
@@ -9,28 +9,28 @@ namespace Azure.AI.TextAnalytics
 {
     /// <summary>
     /// The result of the recognize PII entities operation on a single
-    /// document, containing a collection of the <see cref="CategorizedEntity"/>
+    /// document, containing a collection of the <see cref="PiiEntity"/>
     /// objects containing Personally Identifiable Information that were
     /// found in that document.
     /// </summary>
     public class RecognizePiiEntitiesResult : TextAnalyticsResult
     {
-        internal RecognizePiiEntitiesResult(string id, TextDocumentStatistics statistics, IList<CategorizedEntity> entities)
+        internal RecognizePiiEntitiesResult(string id, TextDocumentStatistics statistics, IList<PiiEntity> entities)
             : base(id, statistics)
         {
-            CategorizedEntities = new ReadOnlyCollection<CategorizedEntity>(entities);
+            PiiEntities = new ReadOnlyCollection<PiiEntity>(entities);
         }
 
         internal RecognizePiiEntitiesResult(string id, string errorMessage)
             : base(id, errorMessage)
         {
-            CategorizedEntities = Array.Empty<CategorizedEntity>();
+            PiiEntities = Array.Empty<PiiEntity>();
         }
 
         /// <summary>
-        /// Gets the collection of named entities containing Personally
+        /// Gets the collection of PII entities containing Personally
         /// Identifiable Information in the input document.
         /// </summary>
-        public IReadOnlyCollection<CategorizedEntity> CategorizedEntities { get; }
+        public IReadOnlyCollection<PiiEntity> PiiEntities { get; }
```

Comment:
I left a comment about this in the Teams channel, but are we set on renaming this key to `PiiEntities`? Seems fine to do so just want to make sure JS/TS are aligned as well.

Reply:
will rename to `entities`